src/prob/tp_opf.jl

```diff
@@ -29,14 +29,18 @@ function post_tp_opf(pm::GenericPowerModel)
         PMs.variable_dcline_flow(pm, cnd=c)
     end
 
+    add_arcs_trans!(pm)
```

Comment:
All `ref` extensions should come at the top of the model, before variables, constraint, or the objective.